src/main/webapp/script.js

```diff
@@ -10,4 +10,75 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License.
\ No newline at end of file
+// limitations under the License.
+
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(getChart);
+
+const CHART_WIDTH = 800;
+const CHART_HEIGHT = 400;
+
+function getYouTubeComments() { 
+  const urlInput = document.getElementById('url-entry');
+  var url = cleanseUrl(urlInput.value);
+  fetch("/YouTubeComments?url="+url)
+      .then(response => response.json()).then((comments) =>{
+        return comments;
```

Comment:
Add a TODO for me to call the getCharts() function

Reply:
Wouldn't getCharts() be calling the getYouTubeComments()?